### PR TITLE
(node/fiber-spec01.tu.lsst.org) EL9 setup

### PR DESCRIPTION
**Pull request by dtapiacl**

### CODE REVIEW

File: hieradata/node/fiber-spec01.tu.lsst.org.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+      ethernet: {}
+      bridge:
+        stp: false
+      ipv4:
+        address1: "140.252.147.137/27"
+        method: "manual"
+      ipv6:
+        method: "disabled"
+      proxy: {}
+network::mroutes_hash:
```
Comment (jhoblitt) [inline hieradata/node/fiber-spec01.tu.lsst.org.yaml]: The `example42/network` module isn't included on EL9 and thus this key won't do anything.

**Review by jhoblitt — CHANGES_REQUESTED** [review]
Please remove the noop hiera key and/or add the nm route configuration if it is needed.

File: hieradata/node/fiber-spec01.tu.lsst.org.yaml (hunk 1 above)
Comment (dtapiacl) [reply hieradata/node/fiber-spec01.tu.lsst.org.yaml]: You are right, it is only supported up to RHEL8. I will just remove it for now as this is just a test device for Parker.

**Review by jhoblitt — APPROVED** [review]
Looks good now. Thanks.

**Pull request merged**